### PR TITLE
Improve docs for Task mfa field

### PR DESCRIPTION
**Pull request by eksperimental**
Related PR: https://github.com/elixir-lang/elixir/pull/11794

### CODE REVIEW

File: lib/elixir/lib/task.ex [hunk 1]
```diff
@@ -225,6 +225,9 @@ defmodule Task do
 
   It contains these fields:
 
+    * `mfa` - a three-element tuple containing the module, function name,
+        and arity; stored for reflection purposes
```
Comment (eksperimental) [inline lib/elixir/lib/task.ex:229]: I'm  also wondering if we should add a mention like this:

> See `Task.async/1` and `Task.async/3` to see how this field is set.

Comment (josevalim) [reply lib/elixir/lib/task.ex:229]:
```suggestion
        and arity invoked to start the task in `async/1` and `async/3`
```

File: lib/elixir/lib/task.ex [hunk 2]
```diff
@@ -233,19 +236,19 @@ defmodule Task do
     * `:owner` - the PID of the process that started the task
 
   """
-  @enforce_keys [:pid, :ref, :owner, :mfa]
+  @enforce_keys [:mfa, :pid, :ref, :owner]
```
[the remark this answers is not in the document]
Comment (eksperimental) [reply lib/elixir/lib/task.ex:239]: That's what I've been doing for the other structs. As the structs grow it's less error prone when you need to visually review code. Additionally ExDoc sorts alphabetically the type specs defined a few lines bellow.

File: lib/elixir/lib/task.ex (hunk 1 above)
Comment (eksperimental) [reply lib/elixir/lib/task.ex]: That is prefect!

File: lib/elixir/lib/task.ex (hunk 2 above)
[the remark this answers is not in the document]
Comment (josevalim) [reply lib/elixir/lib/task.ex:239]: Yeah, so let's go alphabetical here.

File: lib/elixir/lib/task.ex [hunk 3]
```diff
@@ -225,27 +225,30 @@ defmodule Task do
 
   It contains these fields:
 
+    * `mfa` - a three-element tuple containing the module, function name,
+        and arity invoked to start the task in `async/1` and `async/3`
```
Comment (josevalim) [inline lib/elixir/lib/task.ex:229]:
```suggestion
    * `:mfa` - a three-element tuple containing the module, function name,
      and arity invoked to start the task in `async/1` and `async/3`
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**